test/src/test/java/org/apache/accumulo/test/util/CertUtilsTest.java

```diff
@@ -94,22 +98,24 @@ public void createPublicSelfSigned() throws Exception {
   @Test
   public void createSigned() throws Exception {
     CertUtils certUtils = getUtils();
-    File tempSubDir = tempDir.toPath().resolve(testName()).toFile();
-    assertTrue(tempSubDir.isDirectory() || tempSubDir.mkdir());
-    File rootKeyStoreFile = tempSubDir.toPath().resolve("root.jks").toFile();
-    certUtils.createSelfSignedCert(rootKeyStoreFile, "test", PASSWORD);
-    File signedKeyStoreFile = tempSubDir.toPath().resolve("signed.jks").toFile();
-    certUtils.createSignedCert(signedKeyStoreFile, "test", PASSWORD,
-        rootKeyStoreFile.getAbsolutePath(), PASSWORD);
+    Path tempSubDir = tempDir.resolve(testName());
+    if (!Files.isDirectory(tempSubDir)) {
+      Files.createDirectories(tempSubDir);
+    }
```

Comment:
could add similar assertion: `assertTrue(Files.isDirectory(dir))` or move to util method I mentioned earlier